config/install/migrate_plus.migration.custom_peertube_migration.yml

```diff
@@ -0,0 +1,69 @@
+langcode: en
+status: true
+dependencies: { }
+id: custom_peertube_migration
+label: 'Extract peertube media to file'
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_group: custom_peertube_migration
+
+source:
+  plugin: content_entity:media
+  entity_type: media
+  bundle: remote_video
```

Comment:
This doesn't seem to be effective for filtering, so `skip_bundle` is needed later.